yt_dlp/extractor/youtube.py

```diff
@@ -3785,7 +3839,11 @@ def _extract_player_responses(self, clients, video_id, webpage, master_ytcfg, sm
             if 'web' in clients:
                 experiments = traverse_obj(master_ytcfg, (
                     'WEB_PLAYER_CONTEXT_CONFIGS', ..., 'serializedExperimentIds', {lambda x: x.split(',')}, ...))
-                if all(x in experiments for x in self._POTOKEN_EXPERIMENTS):
+                if (
```

Comment:
We can probably remove this initial logic here, since web will always require PO Token now

Keep it consistent with android client and mark the formats as broken unless PO token provided

Reply:
I think that's fair. I would assume the experiment IDs will also differ for other clients.

Reply:
the webpage data will always be `WEB` client right?

Reply:
Yep